client/components/resource-details/SubmitReview.tsx

```diff
@@ -0,0 +1,398 @@
+  border-radius: 5px;
+  outline: none;
+  padding: 7px;
+  font-family: ${fontInter};
+  font-size: 1em;
+`
+
+const AboutYou = styled.h3`
+  text-align: left;
+  width: 100%;
+  margin-left: 10px;
+  margin-right: 10px;
+`
+
+const CancelButton = styled.button`
+  background: white;
+  border: medium solid #ff6961;
+  border-radius: 5px;
+  font-family: ${fontInter};
+  font-weight: ${fontWeight.bold};
+  color: #ff6961;
+  width: 100px;
+  padding: 5px 20px;
+  font-size: ${desktopFontSize.body1};
+  margin-right: 20px;
+  cursor: pointer;
+
+  @media ${smallerThan(width.mobileS)} {
+    margin-bottom: 10px;
+  }
+
+  &:hover {
+    background: #ff6961;
+    color: white;
+  }
+`
+
+const PostButton = styled.button`
+  background: ${colours.primary2};
+  border-radius: 5px;
+  font-family: ${fontInter};
+  font-weight: ${fontWeight.bold};
+  color: white;
+  width: 100px;
+  padding: 5px 20px;
+  border: medium solid ${colours.primary2};
+  font-size: ${desktopFontSize.body1};
+  cursor: pointer;
+
+  @media ${smallerThan(width.mobileS)} {
+    margin-bottom: 10px;
+  }
+
+  &:hover {
+    background: white;
+    color: ${colours.primary2};
+  }
+`
+
+const ButtonDiv = styled.div`
+  display: flex;
+  justify-content: flex-end;
+
+  margin-left: 10px;
+  margin-right: 10px;
+`
+
+const DropdownRow = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-left: 10px;
+  margin-right: 10px;
+`
+
+const Dropdown = styled.select`
+  padding: 5px;
+  border-radius: 5px;
+  background: white;
+  font-family: ${fontInter};
+  &:hover {
+    background: ${colours.neutralLight1};
+  }
+`
+
+const DropdownLabel = styled.label`
+  text-align: left;
+  margin-right: 10px;
+`
+
+const CloseButton = styled.svg`
+  fill: rgb(100, 100, 100);
+  cursor: pointer;
+  &:hover {
+    fill: rgb(0, 0, 0);
+  }
+`
+
+const ModalContent = ({
+  onRequestClose,
+  resourceSlug,
+}: {
+  onRequestClose: () => void
+  resourceSlug: string
+}) => {
+  const [comment, setComment] = useState('')
+
+  const initialRatings: Ratings = RatingCriteria[resourceSlug].reduce(
+    (o, key) => ({ ...o, [key]: 0 }),
+    {}
+  )
+  const [ratings, setRatings] = useState<Ratings>(initialRatings)
+
+  const changeRatings = (label: string, score: number) => {
+    setRatings({ ...ratings, [label]: score })
+  }
+
+  const changeComment = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
+    setComment(e.target.value)
+  }
+
+  const [aboutYou, setAboutYou] = useState<AboutYouData>({
+    faculty: Faculty.Mathematics,
+    status: Status.Other,
+  })
+
+  const changeFaculty = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    setAboutYou({ ...aboutYou, faculty: Faculty[e.target.value] })
+  }
+
+  const changeStatus = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    setAboutYou({ ...aboutYou, status: Status[e.target.value] })
+  }
+
+  const onRequestPost = () => {
+    Object.keys(ratings).map((key) => (ratings[key] *= 10))
```

Comment:
We need to validate mandatory input here. Can address in the next PR though.

Reply:
TBD in next PR